Makefile

```diff
@@ -66,6 +66,11 @@ upload:
 	svn import "$(ARCHIVE_DIR)/$(EXPORT_WIN32)" "$(REPO_TAGS)/archives/$(EXPORT_WIN32)" \
 		-m "added released win32 installer for version $(VERSION)"
 
+.PHONY: pycam/Version.py
+pycam/Version.py:
```

Comment:
Using a file as a target (and marking it as PHONY at the same time) may be a bit confusing for the reader.
What do you think about this one?
```
.PHONY: update-version
update-version:
	echo "VERSION = '$(shell scripts/get-version)'" >pycam/Version.py
```